docs/api/kinematics/rotation.md

```diff
@@ -0,0 +1,189 @@
+# wave.kinematics.rotation
+
+Our rotation library component internally uses the Kindr library's rotation class to perform elementary operations. For more information on the Kindr library rotations, see their documentation [here](http://docs.leggedrobotics.com/kindr/page_rotations.html).
+
+**Class member variables**
+
+- - - -
+
+Internal class storage.  The rotation object of our class internally uses a double precision kindr RotationMatrix.
+
+```
+kindr::RotationMatrixD rotation_object;
+```
+- - - -
+
+The threshold that is used to compare if rotation matrices are sufficiently close to one another.  see isNear() function for more details.
+```
+static double comparision_threshold;
+```
+- - - -
+
+**Class Member Functions**:
+
+## Rotation()
```

Comment:
Hey @chutsu, have you considered documentation from doxygen, etc? Now if Arun changes the interface he will have to manually go through and change everything here even when it's redundant.